Docs 7055 veeva vault documentation edit1 (#592)
mulesoft/docs-connectors: modules/ROOT/pages/veevavault/veevavault-connector-get-started.adoc

> = VeevaVault Connector - Get Started - Mule 4

Reviewer: DONE

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

> Veeva Vault Connector Version 1.1.0

Reviewer: ```suggestion
VeevaVault Connector Version 1.1.0
```
right? one word for the connector name only?

Anypoint Connector for Veeva Vault (VeevaVault Connector) enables you to connect Mule flows to the Veeva Vault platform using the Veeva Vault REST API.

> This connector works with the https://developer.veevavault.com/api/19.1/#authentication[Veeva Vault REST, Bulk, and Streaming API], depending on the operations you configure. Each API call uses a JSON/CSV request and returns JSON input stream responses over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.

Reviewer: GLOBALLY avoid using / to mean "and/or." Choose the most precise conjunction and use it instead.
```suggestion
This connector works with the https://developer.veevavault.com/api/19.1/#authentication[Veeva Vault REST, Bulk, and Streaming APIs], depending on the operations you configure. Each API call uses a JSON or CSV request and returns JSON input stream responses over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.
```

> You can use VeevaVault Connector in Mule applications as an outbound connector with CRUD (Create, Retrieve, Update, Delete), Query, Export, and Spark Validator operations on the following Veeva Vault objects:

Reviewer: ```suggestion
You can use VeevaVault Connector in Mule applications as an outbound connector with CRUD (create, retrieve, update, and delete), query, export, and Apache Spark validation operations on the following Veeva Vault objects:
```

Reviewer: I recommend GLOBALLY lowercasing these operation types, except for the proper noun Spark.
I will stop marking this here, to avoid repetition.

* Documents: Create, Retrieve, Update, Delete, Export
* Document Renditions: Create, Retrieve, Delete
* Vault Object: Create, Retrieve, Update, Delete
* Picklists: Retrieve
* Audit Trail: Retrieve

> * Query: VQL Query to query on Documents, Vault Object, Workflows

Reviewer: ditto lowercase except proper nouns
```suggestion
* Query: VQL query to query on documents, vault object, workflows
```

> * Spark Validator: Validates and verifies the Spark message with signature that is triggered from Veeva Vault

Reviewer: I ran an internet search on spark validator, and found mostly lowercase "validator."
Spark, of course, is init. capped as a proper noun owned by Apache.

> You can configure VeevaVault Connector using Basic Authentication with credentials to connect to a Veeva Vault instance and perform operations.

Reviewer: In this case, I know that Basic Authentication is the name of a Mule plan or contract, but it would not matter if it were lowercased here (as a type of auth) alternatively.
I know these are just pickyisms. Our audience won't care. But I would like to follow common noun and proper noun rules...when we can determine if something really is a name or just a thing...sigh....

== About Connectors

> Anypoint connectors are Mule runtime engine (Mule) extensions that enable you to connect to APIs and resources on external systems, such as Salesforce, Veeva Vault, ServiceNow, Twitter, and so on.

Reviewer: ```suggestion
Anypoint Connectors are Mule runtime engine (Mule) extensions that enable you to connect to APIs and resources on external systems, such as Salesforce, Veeva Vault, ServiceNow, Twitter, and so on.
```
